riscv/src/continuations.rs

```diff
@@ -216,6 +216,7 @@ pub struct DryRunResult<F: FieldElement> {
 /// - The number of rows after which the prover should jump to the shutdown routine.
 pub fn rust_continuations_dry_run<F: FieldElement>(
     pipeline: &mut Pipeline<F>,
+    field: KnownField,
```

Comment:
can get `KnownField` from `F`

Reply:
yea but I wanted to keep them decoupled on purpose, in case we manage to remove F here. But I'll remove it

Reply:
ah this one